### PR TITLE
ipatests: Add PTR record for IP SAN

### PR DESCRIPTION
**Pull request by mrizwan93**
If PTR record is missing for an IP address then cert request
with SAN option throws an error. This fix is to add the PTR
record so that cert request doesn't throw an error.

Signed-off-by: Mohammad Rizwan <myusuf@redhat.com>

### CODE REVIEW

**Comment by mrizwan93** [comment]
This issue found while running test on RHEL machine. Hence no pagure ticket.

File: ipatests/test_integration/test_cert.py [hunk 1]
```diff
@@ -96,6 +99,18 @@ def test_ipa_getcert_san_aci(self):
         certfile = '/etc/pki/tls/certs/test2.pem'
 
         tasks.kinit_admin(self.master)
+
```
Comment (kaleemsiddiqu) [inline ipatests/test_integration/test_cert.py:102]: Why we are just logging the failure from reverse zone addition if any? It should mark the test fail and move to next test.
If this section of test just needs to be executed on a particular platform then we can use tasks.get_platform(self.master) to run this on that particular platform.

**Comment by mrizwan93** [comment]
When I forced `--reverse-zone`, am getting below error. So reverting to initial settings and adding reverse zone as a part of test.
```
2020-08-19 13:33:07,500    DEBUG  [ipatests.pytest_ipa.integration.host.Host.master.IPAOpenSSHTransport] RUN ['ipa', 'dnsrecord-show', 'ipa.test', 'client0']
2020-08-19 13:33:07,500    DEBUG  [ipatests.pytest_ipa.integration.host.Host.master.cmd53] RUN ['ipa', 'dnsrecord-show', 'ipa.test', 'client0']
2020-08-19 13:33:07,500    DEBUG  [ipatests.pytest_ipa.integration.host.Host.master.cmd53] ipa: ERROR: client0: DNS resource record not found
2020-08-19 13:33:07,500    DEBUG  [ipatests.pytest_ipa.integration.host.Host.master.cmd53] Exit code: 2
2020-08-19 13:33:07,500    DEBUG  ipa: ERROR: stderr: ipa: ERROR: client0: DNS resource record not found
2020-08-19 13:33:07,500    DEBUG  
```
http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/721d6eca-e21d-11ea-895a-fa163ee1d49c/runner.log.gz

**Comment by kaleemsiddiqu** [comment]
LGTM, please remove temp commit.

**Comment by mrizwan93** [comment]
Temp commit result: http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/4cdc6b94-e22b-11ea-8ebd-fa163e2b3f63

**Comment by mrizwan93** [comment]
Removing ack for now as I am doing testing with on rhel system.

**Comment by kaleemsiddiqu** [comment]
> Removing ack for now as I am doing testing on rhel system.

@mrizwan93 
Please share the results from rhel system execution.

**Comment by mrizwan93** [comment]
> > Removing ack for now as I am doing testing on rhel system.
> 
> @mrizwan93
> Please share the results from rhel system execution.
Here is the result (report.html)
https://pastebin.ubuntu.com/p/KbZWRnscSF/

**Comment by kaleemsiddiqu** [comment]
@mrizwan93 
Please remove temp commit.

**Comment by abbra** [comment]
master:

* b3d7a70ee00bb5e8a0f1781a378dc169ee455624 ipatests: Add PTR record for IP SAN
* bddbfb79e4749713dd2b22445fb95e3e53d080e8 ipatests: add --skip-overlap-check option to prepare_reverse_zone()
* 1abeb85fbadfaff66694aedc9abfe9229cd3c5d8 PEP8 fixes

**Pull request closed without merging**